terraform/azure-postgresql-flexible-server/bind/main.tf

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Pivotal Software, Inc.
```

Comment:
If you're going to add a copyright statement (which is another debate) it should have the correct year and legal entity.

Reply:
It was added because of the magic of copy-paste. I will change it to be Broadcom Inc.